### ISSUE TITLE
Document Data Refresh Button Update From Splash Page

### ISSUE THREAD

**Issue by anthonycpichardo** [lzim/teampsd#1038]
Remove references in documentation that ask user to click the data refresh button on mtl.how/data.
 

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by staceypark** [comment]
@anthonycpichardo how (if at all) is the ask in this issue different than #1045 ?

**Comment by anthonycpichardo** [comment]
One and the same. Closing this issue.

Duplicate of #1045
